docs/content/doc/installation/with-docker-rootless.en-us.md

```diff
@@ -0,0 +1,337 @@
+---
+date: "2020-02-09T20:00:00+02:00"
+title: "Installation with Docker (rootless)"
+slug: "install-with-docker-rootless"
+weight: 10
+toc: true
+draft: true
+menu:
+  sidebar:
+    parent: "installation"
+    name: "With Docker Rootless"
+    weight: 10
+    identifier: "install-with-docker-rootless"
+---
+
+# Installation with Docker
```

Comment:
As most of these instructions overlap with regular docker image, perhaps instead of this page, a small section in regular docker page with details of what rootless image is, and how to get it starting at 1.12.0-dev(by appending -rootless to tag name)

Reply:
Possible but some option/env are not anymore used like USER_UID and USER_GID.
I was thinking it might be clearer to use a dedicated page that will replace the other one when becoming default.